FRPs/FRP-12.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 12
+title: MEV in l2 & sidechains exploration
+team: @obadiaa @fxfactorial
+created: 2021-06-04
+status: pending
+---
+
+# MEV in l2 & sidechains exploration
```

Comment:
General comment on FRP-12: intro looks great, I'd try to bring a bit more order to the questions/points to research, list format might work but I'd try to order/group them a bit so that there's more of a progression, they seem to be a bit like loose points now